<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopEntryDialog</class>
 <widget class="QDialog" name="DesktopEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset theme="application-x-desktop">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="whatsThis">
        <string>Specific name of this launcher.</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="whatsThis">
        <string>A short description or generic name, like &quot;Web Browser&quot; or &quot;Home Directory&quot;.

It can be left empty.</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="commentEdit">
       <property name="whatsThis">
        <string>A short comment.

It can be left empty.</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="commandLabel">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="commandEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="commandButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="iconEdit">
         <property name="whatsThis">
          <string>Icon to display in file manager, menus, etc.</string>
         </property>
         <property name="text">
          <string notr="true">application-x-desktop</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="iconButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="terminalLabel">
       <property name="toolTip">
        <string>Run in terminal?</string>
       </property>
       <property name="text">
        <string>Terminal:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="terminalCombo">
       <property name="toolTip">
        <string>Run in terminal?</string>
       </property>
       <property name="whatsThis">
        <string>Whether the launched program should run in a terminal window.</string>
       </property>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yes</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="typeCombo">
       <property name="whatsThis">
        <string>Whether this is a link or executes a command.</string>
       </property>
       <item>
        <property name="text">
         <string>Application</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Link</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="catLabel">
       <property name="text">
        <string>Categories:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="catEdit">
       <property name="whatsThis">
        <string>Semicolon separated categories in which the entry should be shown if it is used by the DE's main menu (e.g., when put inside ~/.local/share/applications).

It is needed only when you want to use the created file in the main menu. Otherwise, you could leave it empty.

Examples: AudioVideo, Audio, Video, Development, Education, Game, Graphics, Network, Office, Settings, System, Utility, Qt.</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DesktopEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DesktopEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
